app/assets/javascripts/audit.js

```diff
@@ -0,0 +1,69 @@
+$(document).ready(function () {
```

Comment:
please don't add new JS code into the asset pipeline, new code should live under the webpack directory. (this way you can also write tests and use ES6 features).

Ping me on IRC if you need assistance.

on a similar note, patternfly-react has a built in list-view, maybe it would be easier to use that then to write and maintain all of the JS ourselves?

Reply:
(an example of how to use listview via patternfly-react can be seen [here](https://rawgit.com/patternfly/patternfly-react/gh-pages/index.html?selectedKind=patternfly-react%2FContent%20Views%2FList%20View&selectedStory=List%20of%20expandable%20items&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs)